### ISSUE TITLE
The original tracker for BDD100K MOT as mentioned in the article. 

### ISSUE THREAD

**Issue by HELLORPG** [siyuanliii/masa#51]
Thanks for your excellent and useful repo. 

I have observed that [the tracker used by BDD100K](https://github.com/siyuanliii/masa/blob/main/masa/models/tracker/masa_bdd_tracker.py) has a lot of additional processing compared to [the TAO tracker](https://github.com/siyuanliii/masa/blob/main/masa/models/tracker/masa_tao_tracker.py).  In addition, in the current BDD100K inference processing, it seems that **the similarity calculation method shown in equation 2 in section J.4 is not used, but bi-softmax is directly employed**. I would like to ask:
1. Could you provide an inference configuration file or tracker file that uses the process described in the paper for BDD100K?
2. The results in [model zoo](https://github.com/siyuanliii/masa/blob/main/docs/model_zoo.md) on BDD100K have a significant improvement over the results in the paper. Is this because different inference method and settings is used compared to the paper?

Thank you very much for your attention to these issues. I look forward to your response and appreciate your assistance.

**Comment by siyuanliii** [comment]
Thank you for your detailed observations and questions!
	1.	Inference Method in BDD100K: we follow the approach used in QDTrack, directly employing bi-softmax instead of calculating the average of bi-softmax and cosine similarity as described in Equation 2 of Section J.4 (this is for TAO), as we find this gives slightly better results for BDD100K. However, the tracker parameters already include an option to compute also the cosine similarity. This can be easily modified in the tracker settings. 
	2.	Improved Results in the Model Zoo:
Regarding the better performance compared to the results in the paper, this is partly due to re-training the model using the updated codebase. During training, we sample raw images in a bootstrapping manner, which can lead to slight variations in performance. The overall performance remains stable. If you retrain on your own, it’s unlikely to see significantly better or worse results.
I hope this clarifies your concerns. Please let me know if you need further assistance or additional details!

**Comment by HELLORPG** [comment]
Thank you for your response! Your explanation has clarified my doubts. 🤗
